packages/client/src/test.ts

```diff
@@ -0,0 +1,74 @@
+import { BridgeClient, BridgeConfig } from "./client";
+import readline from "readline";
+import { TransactionSkeletonType } from "@ckb-lumos/helpers";
+
+const myConfig: BridgeConfig = {
+  SIGHASH_DEP: {
+    out_point: {
+      tx_hash: "0xace5ea83c478bb866edf122ff862085789158f5cbff155b7bb5f13058555b708",
+      index: "0x0",
+    },
+    dep_type: "dep_group",
+  },
+  BRIDGE_DEP: {
+    out_point: {
+      tx_hash: "0xa3c3d96042f23411186456f53356dc1c522df92a12814d19c322cce39b5be2fd",
+      index: "0x0",
+    },
+    dep_type: "code",
+  },
+  DEPOSIT_DEP: {
+    out_point: {
+      tx_hash: "0x30ce0f1f27112b781a1eb5951d4f5fee4bda0478cfdda767341f9fa6cf56f49d",
+      index: "0x0",
+    },
+    dep_type: "code",
+  },
+  ANYONE_CAN_PAY_SCRIPT: {
+    code_hash: "0xe683b04139344768348499c23eb1326d5a52d6db006c0d2fece00a831f3660d7",
+    hash_type: "type",
+    args: "0x",
+  },
+  DEPOSIT_CODE_HASH: "0xd7aa21f5395d0bb03935e88ccd46fd34bd97e1a09944cec3fcb59c340deba6cf",
```

Comment:
Are this values for the Testnet? 
Can we have a deployment comment that mentions all the hashes that need to be replaced in this file?